wemake_python_styleguide/violations/best_practices.py

```diff
@@ -1,13 +1,12 @@
 """
 These checks ensure that you follow the best practices.
 
-The source for these best practices is hidden inside countless hours
+The source for these best practices are inside countless hours
 we have spent debugging software or reviewing it.
 
 How do we find inspiration for new rules?
 We find some ugly code during code reviews and audits.
-Then we forbid to use this bad code forever.
-So, this error will never return to our codebase.
+Then we forbid to use those code forever.
```

Comment:
"this code" or "that code".
"those code" is not grammatically correct, (although "those codes" would be, though incorrect for this context).